src/main/resources/templates/pps/home.html

```diff
@@ -14,105 +14,97 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl" id="main-heading">
-                    Pay a penalty for filing your company accounts late
+                    Pay a penalty to Companies House
                 </h1>
-                <p class="govuk-body-l">Use this service to pay
-                    a company's late filing penalty.</p>
-
+                <p class="govuk-body-l">Use this service to make an online payment by credit or debit card if you've
```

Comment:
The class here might need to change to be `p class="govuk-body"`? - referring to the v9 prototype, see https://github.com/companieshouse/sanctions-payment-prototype/blob/main/app/views/v9/01-pay-a-penalty-to-companies-house.html#L46

Reply:
Going to go ahead and change this to "govuk-body" to follow prototype @rbull-ch